backend/src/test/java/gov/cdc/usds/simplereport/api/TranslatorTest.java

```diff
@@ -35,41 +36,60 @@
 class TranslatorTest {
   @Test
   void testEmptyShortDate() {
-    assertNull(parseUserShortDate(""));
+    assertNull(parseUserDate(""));
   }
 
   @Test
   void testNullShortDate() {
-    assertNull(parseUserShortDate(null));
+    assertNull(parseUserDate(null));
   }
 
   @Test
-  void testValidShortDate() {
-    LocalDate result = parseUserShortDate("2/1/2021");
+  void testValidUserDate() {
+    LocalDate result = parseUserDate("2/1/2021");
     assertEquals(2, result.getMonthValue());
     assertEquals(1, result.getDayOfMonth());
     assertEquals(2021, result.getYear());
   }
 
   @Test
   void testValidDateWithLeadingZeros() {
-    LocalDate result = parseUserShortDate("02/01/2021");
+    LocalDate result = parseUserDate("02/01/2021");
     assertEquals(2, result.getMonthValue());
     assertEquals(1, result.getDayOfMonth());
     assertEquals(2021, result.getYear());
   }
 
   @Test
-  void testInvalidShortDate() {
+  void testInvalidDate() {
     IllegalGraphqlArgumentException caught =
         assertThrows(
             IllegalGraphqlArgumentException.class,
             () -> {
-              parseUserShortDate("fooexample.com");
+              parseUserDate("fooexample.com");
             });
     assertEquals("[fooexample.com] is not a valid date", caught.getMessage());
   }
 
+  @Test
+  void testValidShortDate() {
+    LocalDate result = parseUserShortDate("2/1/80");
+    assertEquals(2, result.getMonthValue());
+    assertEquals(1, result.getDayOfMonth());
+    assertEquals(1980, result.getYear());
+  }
+
+  @Test
+  void testInvalidShortDate() {
+    IllegalGraphqlArgumentException caught =
```

Comment:
I see you're changing test names to refer to `ShortDate` if they're testing `parseUserShortDate`. this one looks like it has `ShortDate` in the name but is testing `parseUserDate`

Reply:
actually `testNullShortDate` and `testEmptyShortDate` now test `parseUserDate`. so it seems like sometimes the test names match the method they're testing and sometimes they don't. it makes it a little hard to tell which is intended to be tested

Reply:
This is a good point, thank you! Looking closer at the test cases, I didn't like how any of the naming was done 😆 So changed to hopefully be clearer across the board.